### DISCUSSION TITLE
GERAÇÃO DO PAR DE CHAVES (PRIVADA E PÚBLICA)

### DISCUSSION THREAD

**Discussion by adaopedro** [assoft-portugal/SAF-T-AO#196] [Q&A] [answered]
Olá a todos,

Alguém pode me dizer qual ferramenta está sendo utilizada para gerar a chave privada e por conseguinte a pública? Uma vez que o instrutivo proíbe o uso do OpenSSL para o efeito.

Agraço a atenção,

@cryptolopes 
@IsabelCarvalho77 
@antoniomarinheiro 
@nantonio 
@bacelar-eticadata

**Comment by cryptolopes** [comment, marked as answer]
@AdaoPedro 
Consiste num par de chaves RSA 1024 bits.
O OpenSSL é compatível.

**Reply by adaopedro** [reply]
Grato, @cryptolopes 
Porém, ..
Não sei se conseguiu pegar a premissa da minha pergunta, .. mas prontos,..
Diz-me, por favor: a chave pública gerada por ele (OpenSSL) é válida para submeter junto com o modelo 8?

**Reply by cryptolopes** [reply]
Sim
